packages/domain/package.json

```diff
@@ -68,8 +45,31 @@
     "rimraf": "^3.0.2",
     "ts-jest": "^29.0.3",
     "typescript": "^4.9.3",
-    "tsup": "^6.5.0",
     "wait-for-expect": "^3.0.2"
   },
-  "prettier": "@lens-protocol/prettier-config"
+  "peerDependencies": {
+    "@faker-js/faker": "^7.6.0",
```

Comment:
should all of these be marked as `optional` dependency? 
Otherwise we would get an error on the consumer side when installing the `@lens-protocol/react` that some peers for `domain` package are missing.

Still i think we need to find a better way for mocks that would not force us having `optional` test dependencies in peers as that's confusing from the consumer point of view.

Reply:
Could the middle ground approach for now would be to remove `mocks` from `preconstruct` `entrypoints` and point to the unprocessed file?

mocks/package.json
```
{
  "main": "../src/mocks.ts"
}
```

It should still work properly if we transpile the `@lens-protocol` packages (and we do if you have a look at `transformIgnorePatterns`).

In that case we never publish mocks to the registry and we don't need to have any `peerDependency` to our internal `devDependencies`?
What do you think?